torch/csrc/autograd/function.h

```diff
@@ -84,7 +84,7 @@ TORCH_API void deleteNode(Node* function);
 /// are created in one thread and `C` is created in a new thread, there are *no
 /// guarantees* w.r.t. the ordering of `C` relative to `A` or `B`.
 ///~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-struct TORCH_API Node : std::enable_shared_from_this<Node> {
+struct TORCH_API Node : public std::enable_shared_from_this<Node> {
```

Comment:
Why do we need this change? The default should be public for struct, no?

Reply:
You are right. I'll revert it.